Graphs/VariationTextGraph/styles.js

```diff
@@ -22,6 +22,15 @@ const style = {
         fontSize: "1.2em"
     },
 
+    infoBoxTextNoVariation: {
```

Comment:
create a variable for this and use it in infoBoxText as well

Reply:
@natabal addressed comment, can you pls check